examples/tornjak/README.md

```diff
@@ -35,38 +47,64 @@ kubectl -n spire-server port-forward service/spire-tornjak-backend 10000:10000
 kubectl -n spire-server port-forward service/spire-tornjak-frontend 3000:3000
 ```
 
-You can now access Tornjak at [localhost:3000](http://localhost:3000).
+You can now access Tornjak with your browser at [localhost:3000](http://localhost:3000).
 
 See [values.yaml](./values.yaml) for more details on the chart configurations to achieve this setup.
 
-## Tornjak and Ingress with ingress-nginx
+## Deploy Tornjak with ingress-nginx
 
-Update examples/production/example-your-values.yaml with your information, most importantly, trustDomain.
+Update your-values.yaml with your ingress information, most importantly, trustDomain, and redeploy
+adding the following:
 
 ```shell
-helm upgrade --install --namespace spire-server spire charts/spire \
---values tests/integration/psat/values.yaml \
---values examples/tornjak/values.yaml \
---values examples/tornjak/values-ingress.yaml \
 --set global.spire.ingressControllerType=ingress-nginx \
---render-subchart-notes --debug
+--values examples/tornjak/values-ingress.yaml
 ```
 
-## Tornjak and Ingress on Openshift
+## Deploy Tornjak with Ingress on Openshift
+
+Obtain the OpenShift Apps Subdomain for Ingress and assign it to environment variable:
 
-When deploying on Openshift, follow the deployment setup as described in
-[Openshift README](../openshift/README.md)
+```shell
+export appdomain=$(oc get cm -n openshift-config-managed  console-public -o go-template="{{ .data.consoleURL }}" | sed 's@https://@@; s/^[^.]*\.//')
+echo $appdomain
```

Comment:
Is there anything openshift specific anymore in the instructions? They start now from deploy without tornjak from the main install instructions, which include setting up for openshift. I think we might be able to remove this whole section?

Reply:
Well, we just need to pass the `appdomain` into `trustDomain`